tests/client/test_client_task_meilisearch.py

```diff
@@ -156,3 +156,16 @@ def test_delete_tasks_by_filter(client):
     assert (
         "statuses=succeeded%2Cfailed%2Ccanceled" in tasks_after.results[0].details["originalFilter"]
     )
+
+
+@pytest.mark.usefixtures("create_tasks")
+def test_get_tasks_in_reverse(client):
+    """Tests getting the global tasks list in reverse."""
+    tasks = client.get_tasks({})
+    reverse_tasks = client.get_tasks({"reverse": "true"})
+
+    assert tasks.results[0].uid > tasks.results[-1].uid
+    assert reverse_tasks.results[0].uid < reverse_tasks.results[-1].uid
+
+    assert tasks.results[0] == reverse_tasks.results[-1]
+    assert reverse_tasks.results[0] == tasks.results[-1]
```

Comment:
I think it's safe to simplify this and assume if one check is correct everything is good since as you mentioned Meilisearch itself is doing all the work here and we are just checking that we can send the parameter.

```suggestion
    assert tasks.results[0] == reverse_tasks.results[-1]
```

Reply:
I don't think the only test you highlighted is enough to assert that the parameter is getting passed, if there was only 1 task then it will pass without the parameter.

In any case, I agree with you, I'll force push in a sec.

Reply:
That is true, but I still think we are ok because if the parameter is wrong Meilisearch should return an error.